src/reducers/index.ts

```diff
@@ -1,10 +1,18 @@
 import { combineReducers } from "redux";
+import { Visualization } from "../models/visualization";
+import { vizReducer } from "./viz-reducer";
 
-export interface IState{
-    
+export interface IVizState {
+    visualizations: Visualization[],
+    newVisualization: Visualization
 }
 
+export interface IState{
+    visualizationSet: IVizState
+    newVisualization: IVizState
+}
```

Comment:
You only need one variable, of type IVizState, it will hold visualizations and newVis